APPS-1269 APPS-1270 Update manifest out unqualified file

Gvaihir opened the pull request.

Gvaihir committed:
APPS-1269 APPS-1270 Update manifest handling unqualified file IDs
(feat) apps_1269_1270_unqualified* - integration test for unqualified name handling

Gvaihir committed:
APPS-1269 APPS-1270 Update manifest handling unqualified file IDs
(feat) apps_1269_1270_unqualified* -

Gvaihir committed:
APPS-1269 APPS-1270 Update manifest handling unqualified file IDs
(feat) ApplicationCompiler.scala - Frag applets have "VIEW" access to allProjects. + Tests

Gvaihir committed:
APPS-1269 APPS-1270 Update manifest handling unqualified file IDs
(fix) apps_1269_1270_unqualified_* - fixture update
(fix) build.sbt - dxApi version

Gvaihir committed:
APPS-1269 APPS-1270 Update manifest handling unqualified file IDs
(fix) RELEASE_NOTES.md - update
(fix) run_tests.py - included tests

commandlinegirl reviewed (dismissed):
This LGTM module passing `integration` and `cwl_conformance` tests with updated dxApiVersion. Thank you!

Gvaihir committed:
Merge branch 'develop' into APPS-1269_APPS-1270_manifest_out_unqualified_file_id

# Conflicts:
#	RELEASE_NOTES.md

Gvaihir committed:
APPS-1269 APPS-1270 Update manifest handling unqualified file IDs
(fix) apps_1269_1270_* - added dx:// prefix, which does not change the test outcome

Gvaihir committed:
APPS-1269 APPS-1270 Update manifest handling unqualified file IDs
(fix) apps-788.wdl - change task nam because of possible conflicts when running tests

Gvaihir committed:
Merge branch 'develop' into APPS-1269_APPS-1270_manifest_out_unqualified_file_id
